website/blog/2020-02-26-cobalt.md

```diff
@@ -0,0 +1,447 @@
+---
+author: Tomasz Godzik
+title: Metals v0.8.1 - Cobalt
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We are excited to announce the release of Metals v0.8.1. This release includes a
+large number of bug fixes and some new features.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">313</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">100</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">19</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">52</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">2</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/19?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with help from [Lunatech](https://lunatech.com) along with contributors from the
+community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/) and
+give Metals a try!
+
+- insert missing abstract members
+- new file provider
+- enable rename preview for Visual Studio Code
+- debug and run working in Emacs and soon in Vim
+
+## Insert missing abstract members
+
+Thanks to yet again amazing work by
+[tanishiking](https://github.com/tanishiking) we now support the quick fix code
+action for implementing abstract class members. This code action is available in
+case of errors and works in the same principle as the recent completions. It
+will add any missing abstract members at the top of the class with the default
+implementation of ???.
+
+![action](https://user-images.githubusercontent.com/9353584/73630120-8b0bc880-4698-11ea-8370-32b1fb2f86fd.gif)
+
+## New file provider
+
+In the last release we added a support for worksheets, which are an easy way to
+quickly evaluate some code. To create a new worksheet you need to create a file
+with the extension `.worksheet.sc`, which was not that easy to figure out. To
+fix that situation, thanks to
+[alekseiAlefirov](https://github.com/alekseiAlefirov), we now have a menu to
+create different types of new files including classes, traits, object, package
+objects and of course worksheets. At this time it's only available in Visual
+Studio Code extension, but will soon be included also in the coc-metals vim
+extension.
+
+![new-file-provider](https://user-images.githubusercontent.com/10850363/73563467-adfe6880-445d-11ea-89f6-a9a6398034da.gif)
+
+## Enable rename preview for Visual Studio Code
+
+Rename previews were in the last Visual Studio Code release, which enable users
```

Comment:
Thoughts on maybe linking to it in case people aren't sure what we are referring to?

https://code.visualstudio.com/updates/v1_42#_rename-preview